readthedocs/doc_builder/environments.py

```diff
@@ -380,7 +385,11 @@ def get_wrapped_command(self):
 
     def _escape_command(self, cmd):
         r"""Escape the command by prefixing suspicious chars with `\`."""
-        return self.bash_escape_re.sub(r'\\\1', cmd)
+        command = self.bash_escape_re.sub(r"\\\1", cmd)
+
+        # HACK: avoid escaping `$READTHEDOCS_OUTPUT` variable
+        command = command.replace("\\$READTHEDOCS_OUTPUT", "$READTHEDOCS_OUTPUT")
+        return command
```

Comment:
This logic is expanded in https://github.com/readthedocs/readthedocs.org/pull/9971/files#diff-12edd53a59d4da0db9b76c984ed4cfdeecaab2d6141bf2eb026ea6ccbaf5c210R385 and I think we should keep it unless we are worried for some specific reason.